src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -97,7 +97,7 @@ public FeedbackResponseCommentAttributes getFeedbackResponseComment(Long feedbac
         
         FeedbackResponseComment frc = getFeedbackResponseCommentEntity(feedbackResponseCommentId);
         
-        if (frc == null) {
+        if (frc == null || JDOHelper.isDeleted(frc)) {
```

Comment:
Isn't this redundant since <code>getFeedbackResponseCommentEntity</code> does this check already?